src/components/Navbar/Navbar.jsx

```diff
@@ -1,78 +1,101 @@
-import {useState} from "react";
-import {Link} from "react-router-dom";
-import { useEffect } from "react/cjs/react.production.min";
+import "./Navbar.css";
+import { useState } from "react";
+import { useNavigate } from "react-router";
+import { Link } from "react-router-dom";
+import { useAuth } from "../../hooks";
 
 const Navbar = () => {
-    const [menuOpen,setMenuOpen] = useState(false);
+  const navigate = useNavigate();
+  const { authState, authDispatch } = useAuth();
+  const [menuOpen, setMenuOpen] = useState(false);
+  const userName = authState.user;
 
-    const openMenuBar=()=>{
-      setMenuOpen(true);
-    }
-    const closeMenuBar=()=>{
-      setMenuOpen(false);
-    }
-    
-    return (
-      <header>
-        <div className="nav-header">
-          <div className="title">
-            <i className="fas fa-bars menu-bar" onClick={openMenuBar}></i>
-            <Link to="/">
-              <h2 className="nav-title">CheckMate</h2>
-            </Link>
-          </div>
-          <div className="search">
-            <span className="btn-search">
-              <i className="fas fa-search"></i>
-            </span>
-            <input type="text" placeholder="Search" className="input-search" />
-          </div>
-          <div className="user-controls">
-            <Link to="/login">
-              <button className="btn-login">Login</button>
-            </Link>
-            <Link className="btn-check" to="/wishlist">
-              <i className="far fa-heart"></i>
-              <span className="count">0</span>
-            </Link>
-            <Link className="btn-check" to="/cart">
-              <i className="fas fa-shopping-cart"></i>
-              <span className="count">0</span>
-            </Link>
+  const openMenuBar = () => {
+    setMenuOpen(true);
+  }
+  const closeMenuBar = () => {
+    setMenuOpen(false);
+  }
+
+  const checkStatus = (userName) => {
```

Comment:
The name could be better I guess. Maybe something like `checkAction` would be better in my opinion.